# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: tools/profiler/gecko/nsProfiler.cpp
# Commit: 3ea11e90d26d
# Full Hash: 3ea11e90d26dd8974b3da41335f5861c491b0c90
# Author: Nazım Can Altınova <canaltinova@gmail.com>
# Date: 2024-09-17 09:28:38
# Regressor Bug: 1634785
# File Overlap Count: 1
# Description:
#   Bug 1634785 - Rename BaseProfilerSharedLibraries.h to SharedLibraries.h r=mstange,profiler-reviewers
#   
#   Now that we only have one implementation of this code, we don't have to
#   differentiate it with "BaseProfiler".
#   
# ==============================================================================

diff -r cb492d775d37 -r 3ea11e90d26d tools/profiler/gecko/nsProfiler.cpp
--- a/tools/profiler/gecko/nsProfiler.cpp	Mon Sep 16 21:09:34 2024 +0000
+++ b/tools/profiler/gecko/nsProfiler.cpp	Mon Sep 16 21:09:34 2024 +0000
@@ -37,7 +37,7 @@
 #include "nsString.h"
 #include "nsThreadUtils.h"
 #include "platform.h"
-#include "BaseProfilerSharedLibraries.h"
+#include "SharedLibraries.h"
 #include "zlib.h"
 
 #ifndef ANDROID